.cookiecutter-replay.json

```diff
@@ -0,0 +1,46 @@
+{
+  "cookiecutter": {
+    "Introduction": "",
+    "type": "api",
```

Comment:
This should be `lib` type. You can also compare here: https://github.com/frequenz-floss/frequenz-client-microgrid-python/blob/v0.x.x/.cookiecutter-replay.json